test/visual/components.js

```diff
@@ -3,221 +3,186 @@
 
 /**
  * Customizable test function
- * @componentName component name as specified in styleguide to test
+ * @skip browsers to skip
+ * @componentName the component name to test
+ * @variationName variation name
  * @aptureOrExampleNumber example number (index starts from 1) or array of
  *   selectors for capture-elements
  * @ignoreElements ignore ignoreElements selector (string) or selectors (Array of string)
- * @fn
+ * @fn callback(actions, find) –  Optional callback describes a sequence of actions to bring the
+ * page to this state, starting from a previous state of the suite.
 **/
-function test(componentName, captureOrExampleNumber, ignoreElements, fn = () => {}) {
-  return function (suite) {
+function testVariation(skip, componentName, variationName = 'normal', captureOrExampleNumber = 1,
```

Comment:
The skip parameter should probably be defined in the end and have a default value?